flags/conditions/validators.py

```diff
@@ -0,0 +1,50 @@
+import re
+from distutils.util import strtobool
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.utils import dateparse
+
+
+validate_path = RegexValidator(
+    re.compile(r"^[^\s:?#]+$", re.UNICODE),
+    message=(
+        "Enter a valid path without a URL scheme, query string, or fragment."
+    ),
+    code="invalid",
+)
+
+
+validate_parameter = RegexValidator(
+    re.compile(r"^[-_\w=]+$", re.UNICODE),
+    message="Enter a valid HTTP parameter name.",
+    code="invalid",
+)
+
+
+def validate_boolean(value):
+    message = "Enter one of 'on', 'off', 'true', 'false', etc."
+    try:
+        strtobool(value)
+    except ValueError:
+        # This was a string with an invalid boolean value
+        raise ValidationError(message)
+    except AttributeError:
+        # This was not a string
+        if not isinstance(value, (int, bool)):
+            raise ValidationError(message)
+
+
+def validate_user(value):
+    from django.contrib.auth.models import User
+
+    try:
+        User.objects.get(username=value)
```

Comment:
It might be better to use [`get_user_model`](https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#referencing-the-user-model) here to account for projects that have customized the user model. Although to do that fully you'd have support e.g. alternate values for [`USERNAME_FIELD`](https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#django.contrib.auth.models.CustomUser.USERNAME_FIELD).

Reply:
Good catch! Fixed in 9a351cf.